.github/workflows/action-image-scan-deploy-dashing.yaml

```diff
@@ -0,0 +1,31 @@
+on: 
+  push:
+    branches:
+      - master
+      - main
+
+env:
+  ROS_VERSION: 2
+  UBUNTU_DISTRO: bionic
+  ROS_DISTRO: foxy
```

Comment:
Couldn't we define matrices for UBUNTU_DISTRO and ROS_DISTRO? (by the way, with the proposed update of defining the devel image from the ROS images, I think the UBUNTU_DISTRO will not be required anymore)

Reply:
Concerning the update with ros images, when we do that we do this minor fix. Also, check [here](https://github.com/open-br/ros_ws/pull/13#discussion_r515422612)

Reply:
@caioaamaral is going to do this update, right?